<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>watchdog</class>
 <widget class="QWidget" name="watchdog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1814</width>
    <height>956</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>881</width>
     <height>791</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>code</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>低价买入预警</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>高价卖出预警</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>实时价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>预警是否弹窗</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>time</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>330</y>
     <width>31</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>《</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>910</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>更新监控股票</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>860</y>
     <width>838</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>运行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>终止运行</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>添加监控信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="text">
         <string>完成添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_10">
        <property name="text">
         <string>删除记录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>将表格信息存入csv</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>行情扫描周期</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>60</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>120</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>150</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>210</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>240</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>300</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string> s</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>40</y>
     <width>831</width>
     <height>791</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>转债code</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>转债名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>转债价格</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>转股价</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pb</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>到期时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>溢价率</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>波动率</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>860</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>手动刷新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
